config/docker.database.yml

```diff
@@ -0,0 +1,20 @@
+# This configuration is tailored for use with docker-compose. See DOCKER.md for more information.
+
+development:
+  adapter: postgresql
+  database: openstreetmap
+  username: openstreetmap
+  password: openstreetmap
```

Comment:
If you're using trust auth (which you appear to be) then you shouldn't need a password.